### PR TITLE
Added to provenance and versioning

### PR DESCRIPTION
**Pull request by kaiec**
My first take on provenance and versioning. And my second attempt to send this pull request :-)

### CODE REVIEW

**Comment by mapfde** [comment]
Looks fine to me.

File: CompIndex/CompIndex-stage.txt [hunk 1]
```diff
@@ -96,8 +96,19 @@ B Creating RDF data
 C Generates RDF data from non-RDF sources.
 C Knows methods for generating RDF data from tabular data in formats such as Comma-Separated Values (CSV).
 C Knows methods such as Direct Mapping of Relational Data to RDF (2012) for transforming data from the relational model (keys, values, rows, columns, tables) into RDF graphs.
-B Versioning RDF data
-B RDF data provenance
+B Data Provenance
+C Understands what provenance information is: information about how the data at hand was created (who, where, when, how).
+C Understands the difference between provenance information represented in RDF in contrast to provenance information about RDF data.
+B Data Versioning
+C Understands that versioning may be used to ensure stable, immutable data.
```
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: Versioning does not so much "ensure" immutability as declare an intention.  How about: "Understands that versioning may be used to mark data as stable and immutable."

File: CompIndex/CompIndex-stage.txt [hunk 2]
```diff
@@ -96,8 +96,19 @@ B Creating RDF data
 C Generates RDF data from non-RDF sources.
 C Knows methods for generating RDF data from tabular data in formats such as Comma-Separated Values (CSV).
 C Knows methods such as Direct Mapping of Relational Data to RDF (2012) for transforming data from the relational model (keys, values, rows, columns, tables) into RDF graphs.
-B Versioning RDF data
-B RDF data provenance
+B Data Provenance
+C Understands what provenance information is: information about how the data at hand was created (who, where, when, how).
+C Understands the difference between provenance information represented in RDF in contrast to provenance information about RDF data.
```
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: Or: ...versus information about the provenance of RDF data.

File: CompIndex/CompIndex-stage.txt [hunk 3]
```diff
@@ -96,8 +96,19 @@ B Creating RDF data
 C Generates RDF data from non-RDF sources.
 C Knows methods for generating RDF data from tabular data in formats such as Comma-Separated Values (CSV).
 C Knows methods such as Direct Mapping of Relational Data to RDF (2012) for transforming data from the relational model (keys, values, rows, columns, tables) into RDF graphs.
-B Versioning RDF data
-B RDF data provenance
+B Data Provenance
+C Understands what provenance information is: information about how the data at hand was created (who, where, when, how).
```
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: Or: Understands that provenance information is about how...

File: CompIndex/CompIndex-stage.txt [hunk 4]
```diff
@@ -96,8 +96,19 @@ B Creating RDF data
...
+C Understands the difference between provenance information represented in RDF in contrast to provenance information about RDF data.
+B Data Versioning
+C Understands that versioning may be used to ensure stable, immutable data.
+C Recognizes that versioning may be part of a full data provenance chain.
```
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: "part of a chain" is a bit confusing - not entirely sure what the message is here.

File: CompIndex/CompIndex-stage.txt [hunk 5]
```diff
@@ -96,8 +96,19 @@ B Creating RDF data
...
+B Data Identification
+C Understands that providing information about data requires identification of data using named graphs (-> Link) or by other means.
```
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: Not quite clear.  Does this mean something like: "...requires that the data be (?unambiguously) identified, e.g., by using URIs or named graphs.

File: CompIndex/CompIndex-stage.txt [hunk 6]
```diff
@@ -96,8 +96,19 @@ B Creating RDF data
...
+C Knows VoID as a vocabulary to identify and describe datasets.
+B Representation of Provenance
+C Understands the concept of a provenance chain: data is created from other data which has its own provenance.
+C Knows W3C PROV for the representation of provenance chains for arbitrary resources.
+C Knows provenance information contained in Dublin Core metadata
```
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: Not sure about "contained".  How about: Knows that Dublin Core properties, such as X, inherently carry provenance information.

File: CompIndex/CompIndex-stage.txt (hunk 1 above)
Comment (mapfde) [inline CompIndex/CompIndex-stage.txt]: Yes, this is better.

Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: Approved by WG.

File: CompIndex/CompIndex-stage.txt (hunk 2 above)
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: WG agrees to drop this.

File: CompIndex/CompIndex-stage.txt (hunk 3 above)
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: WG approves.

File: CompIndex/CompIndex-stage.txt (hunk 4 above)
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: WG agrees to drop.

File: CompIndex/CompIndex-stage.txt (hunk 5 above)
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: WG approves:

requires that the data be identified, e.g., by using URIs or named graphs.

**Comment by stuartasutton** [comment]
Looks fine to me as well.

File: CompIndex/CompIndex-stage.txt [hunk 7]
```diff
@@ -96,8 +96,17 @@ B Creating RDF data
...
+C Understands that provenance information is about how the data at hand was created (who, where, when, how).
+B Data Versioning
+C Understands that versioning may be used to mark data as stable and immutable.
+B Data Identification
+C Understands that providing information about data requires that the data be identified, e.g., by using URIs or named graphs.
+C Knows VoID as a vocabulary to identify and describe datasets.
+B Representation of Provenance
+C Understands the concept of a provenance chain: data is created from other data which has its own provenance.
+C Knows W3C PROV for the representation of provenance chains for arbitrary resources.
+C Knows that Dublin Core properties, such as dc:creator, inherently carry provenance information.
```
Comment (tombaker) [inline CompIndex/CompIndex-stage.txt]: Test comment.

**Pull request merged**